packages/shared/types/index.ts

```diff
@@ -138,6 +138,8 @@ export enum FileType {
     WEBP = "webp",
     JPEG = "jpeg",
     SEVEN_Z = "7z",
+    TAR_GZ = "tar.gz", // Possibly Redundant?
```

Comment:
The way file types are resolved currently, no file will ever resolve to this type
so yeah this is redundant

Reply:
I will remove it

Reply:
done